docs/modules/ROOT/nav.adoc

```diff
@@ -39,6 +39,8 @@ endif::env-packaged[]
 .Tutorials
 * xref:tutorial-onpremise.adoc[On-premise example]
 * xref:tutorial-kubernetes.adoc[Kubernetes example]
+* xref:tutorial-standalone.adoc[Standalone example]
+* xref:integrating-with-existing-deployments.adoc[Integrating with existing monitoring pipelines]
```

Comment:
Should this be a tutorial? In my head this would be more of a reference guide, though I can see why you did it this way.

Reply:
Maybe better to be referenced by Standalone example?

Reply:
Not quite sure what you mean? I think this page can live as a top-level nav item, I was more commenting on the fact that it's a tutorial, when users may be best served by a reference guide rather than two tutorials? What do you think?